packages/components/src/components/gif.tsx

```diff
@@ -107,7 +107,7 @@ const Gif = ({
         clearTimeout(hoverTimeout.current!)
         setHovered(true)
         hoverTimeout.current = window.setTimeout(() => {
-            pingback.onGifHover(gif, user, e.target as HTMLElement, attributes)
+            pingback.onGifHover(gif, user?.id, e.target as HTMLElement, attributes)
```

Comment:
We only ever used the user.id, but yet the whole object was passed, now only passing the id